grpc-protobuf/logging.proto

```diff
@@ -40,4 +44,18 @@ message DoguLogMessage {
   google.protobuf.Timestamp timestamp = 1;
   // message contains the content of logged message
   string message = 2;
+}
+
+enum LogLevel {
```

Comment:
Basically, a protobuf describes an API. All things exported should contain a doc comment. Given a sufficient description some exported symbols can be left out, f. i. if `LogLevel` is written like so, I would not add comments on each log level value:

```suggestion
// LogLevel defines values which control the logging in decreasing verbosity.
enum LogLevel {
```

Reply:
Many thanks for the suggestion. I have implemented it.